### PR TITLE
ukrainian translation (skills)

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

**Comment by felix642** [comment]
Hi @burrbull, I see that you added a bunch of #, fuzzy above the translations. This usually means that the translation are not ready yet. Did you mean to do that ?

File: files/lang/uk.po [hunk 1]
```diff
@@ -5221,282 +5293,382 @@ msgid ""
...
 msgid "All of the creatures may offer to join you."
-msgstr ""
+msgstr "Всі істоти можуть захотіти приєднатись до Вас."
 
+#, fuzzy
 msgid ""
 "%{skill} increases your hero's movement points over water by %{count} "
 "percent."
-msgstr ""
+msgstr "содиниці руху героя по воді на %{count} відсотків."
```
Comment (felix642) [inline files/lang/uk.po:5582]: In the msgid there is a %{skill}, but it's not there in the msgstr. Is there a reason ?

**Review by felix642 — CHANGES_REQUESTED** [review]

**Comment by burrbull** [comment]
> Hi @burrbull, I see that you added a bunch of #, fuzzy above the translations. This usually means that the translation are not ready yet. Did you mean to do that ?

I just following @kant2002  recommendation:
https://github.com/ihhub/fheroes2/pull/5406#issuecomment-1131432398

File: files/lang/uk.po (hunk 1 above)
Comment (burrbull) [reply files/lang/uk.po]: No reason. Just typo. Fixed

Comment (felix642) [reply files/lang/uk.po]: Ok that's perfect :) Just wanted to make sure that was not a mistake.

**Review by felix642 — APPROVED** [review]

**Comment by felix642** [comment]
> > Hi @burrbull, I see that you added a bunch of #, fuzzy above the translations. This usually means that the translation are not ready yet. Did you mean to do that ?
> 
> I just following @kant2002 recommendation: [#5406 (comment)](https://github.com/ihhub/fheroes2/pull/5406#issuecomment-1131432398)

Ok that's perfect :) Just wanted to make sure that was not a mistake.

**Comment by burrbull** [comment]
yes, please

**Review by felix642 — APPROVED** [review]

**Comment by ihhub** [comment]
@burrbull , thank you very much for contribution to the project!

**Pull request merged**